extensions/flags/src/Api/Controller/ListFlagsController.php

```diff
@@ -51,11 +52,15 @@ protected function data(ServerRequestInterface $request, Document $document): it
             $include[] = 'post.user.groups';
         }
 
+        /** @var \Illuminate\Database\Eloquent\Collection<int, Flag> $flags */
         $flags = Flag::whereVisibleTo($actor)
-            ->latest('flags.created_at')
-            ->groupBy('post_id')
             ->limit($limit + 1)
             ->offset($offset)
+            ->whenPgSql(
+                fn (Builder $query) => $query->distinct('post_id')->orderBy('post_id'),
+                else: fn (Builder $query) => $query->groupBy('post_id')
+            )
+            ->latest('flags.id')
```

Comment:
Why did we chance the "latest" field? I guess the result will be the same, since things are created with ascending IDs, but is the goal to sort more performantly? I worry that we lose some explicitness here, and if we manually change "created at", sorting would be incorrect.

Reply:
I do not recall, but I think this wasn't supposed to make it to the commit (was battling with tests)

reverting